mindsdb/integrations/handlers/palm_handler/palm_handler.py

```diff
@@ -18,14 +19,37 @@
 )
 
 
+class PalmHandlerArgs(BaseModel):
+    target: str = None
+    model_name: str = "models/chat-bison-001"
+    mode: str = "default"
+    predict_params: dict = None
+    input_text: str = None
+    ft_api_info: dict = None
+    ft_result_stats: dict = None
+    runtime: str = None
+    max_output_tokens: int = 64
+    temperature: float = 0.0
+    api_key: str = None
+    palm_api_key: str = None
+
+    question_column: str = None
+    answer_column: str = None
+    context_column: str = None
+    prompt_template: str = None
+    prompt: str = None
+    user_column: str = None
+    assistant_column: str = None
+
+
```

Comment:
adding the following config class nested under your base model will prevent additional args being passed in query
 
 class Config:
        extra = Extra.forbid